ush/ufsda/r2d2.py

```diff
@@ -2,13 +2,44 @@
 import re
 from solo.configuration import Configuration
 from solo.date import date_sequence, Hour, DateIncrement
+import yaml
+import os
 
 possible_args = [
     'provider', 'experiment', 'database', 'type', 'file_type',
     'resolution', 'model', 'user_date_format', 'fc_date_rendering', 'tile',
 ]
 
 
+def setup(shared_root='', r2d2_config_yaml='r2d2_config.yaml'):
```

Comment:
I think I'd rather have a `r2d2_config_ctest.yaml` or something like that and then set `R2D2_CONFIG=/path/to/r2d2_config_ctest.yaml` at test runtime. I think that's neater than this, especially given the issue of this approach not being viable for different platforms/filesystems for 'real' databases.

Reply:
so keep a setup step that would export R2D2_CONFIG?

Reply:
the runtime env would export it, either through 'sourcing' or module load, etc.

Reply:
I definitely don't like that approach and have been trying to avoid it for a while. How about I keep it that way for now and revisit when closer to actually cycling? It doesn't have to be used. We can remove it later, but I do like an explicit step to setup r2d2. The default could be coming from the runtime env.

Reply:
Yeah we need to discuss this more I think, particularly with @aerorahul. You can keep it for now, yes, but I feel like defining it explicitly leads to 'separate' databases, when we really should only have 1 shared DB per platform and 1 per user for output/archival of experiments.